pallets/pool-registry/src/lib.rs

```diff
@@ -241,6 +254,7 @@ pub mod pallet {
 			currency: T::CurrencyId,
 			max_reserve: T::Balance,
 			metadata: Option<Vec<u8>>,
+			write_off_policy: BoundedVec<WriteOffRuleOf<T>, MaxWriteOffPolicySizeOf<T>>,
```

Comment:
Why, instead of adding here `write_off_policy` as a parameter we are not doing a batch with:
- PoolRegistry::register(..)
- Loans::update_write_off_policy(..)
?

Reply:
The batch transaction will go through CFG governance. Two issues with the above proposal:
1. The origin of the transaction will be `Root` rather than the pool admin, since CFG governance is accepting it. This means we'd need to modify `update_write_off_policy` to be callable also by `Root`, which I would prefer not to.
2. The call will be displayed in governance UIs like Polkadot.js, so keeping it not a batch makes it much more readable.